source

```diff
@@ -102573,11 +102574,12 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    (double-quoted) state</span>. Switch to the <span>character reference state</span>.</dd>
 
    <dt>U+0000 NULL</dt>
-   <dd><span>Parse error</span>. Append a U+FFFD REPLACEMENT CHARACTER character to the current
-   attribute's value.</dd>
+   <dd><span data-x="parse-error-unexpected-null-character">unexpected-null-character</span><span>parse
+   error</span>. Append a U+FFFD REPLACEMENT CHARACTER character to the current attribute's value.</dd>
 
    <dt>EOF</dt>
-   <dd><span>Parse error</span>. Emit an end-of-file token.</dd>
+   <dd>This is an <dfn data-x="parse-error-eof-in-attribute-value">eof-in-attribute-value</dfn>
```

Comment:
We can reuse `eof-in-tag` here. It's not any different than EOF in attribute name.